src/utils/vr-interstitial.js

```diff
@@ -14,7 +14,7 @@ const getUserGesture = () => {
   });
 };
 
-export async function handleExitTo2DInterstitial(isLower, exitAction) {
+export async function handleExitTo2DInterstitial(isLower, exitAction, nonFullscreen) {
```

Comment:
Thoughts on skipping "Continue" when entering the media browser? Iirc you added it so that you had an opportunity to "straighten out your head" after selecting an item -- perhaps we want to only have the "continue" screen on the way back into VR mode.

Reply:
the continue is there because we need a user gesture to transition to full screen mode, so not needed in this case